Source/Tools/FEXServer/Unwind/Unwind_x86.cpp

```diff
@@ -0,0 +1,132 @@
+#include <FEXCore/Core/UContext.h>
+#include <FEXCore/Utils/LogManager.h>
+
+#include "CoreDumpService.h"
+#include "Unwind_x86.h"
+#include "ELFMapping.h"
+#include "FileMapping.h"
+#include <libunwind-x86.h>
+#include <dwarf.h>
+#include <algorithm>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include <cstdint>
+#include <dirent.h>
+#include <fcntl.h>
+#include <filesystem>
+#include <fstream>
+#include <map>
+#include <list>
+#include <unistd.h>
+#include <signal.h>
+#include <string>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <vector>
+
+namespace Unwind::x86 {
+#define panic(args...) \
```

Comment:
`panic` suggests fatal error that causes FEX to exit immediately. Something like `print_error` would be more fitting.

Reply:
Removed the panics and just printed the errors instead, which is the better outcome. Since the rest of the code will safely early exit or continue.